index.bs

```diff
@@ -5798,6 +5803,109 @@ map between [=user contexts=] and [=screen orientation override=].
 
 ### Commands ### {#module-emulation-commands}
 
+#### The emulation.setDisableScriptingOverride Command ####  {#command-emulation-setDisableScriptingOverride}
+
+The <dfn export for=commands>emulation.setDisableScriptingOverride</dfn> command
```

Comment:
At risk of bikeshedding names, what about `emulation.setJavascriptEnabled`? I don't think we need to worry about non-JS languages, and unlike the other commands in this module, this seems more like _configuration_ than providing mock ("override") data (tbh I'd also be OK with it being in the `browsingContext` module, even though you can apply it to user contexts).

Reply:
Agree, but only in case if we agree on `disable: bool / null` instead of `disable: true / null`.

Reply:
I'm not quite sure what's wrong with `setJavascriptEnabled` with a boolean `enabled` that's either `false` or `null`.